src/bin/main.rs

```diff
@@ -3,7 +3,9 @@ use http_server::run;
 #[tokio::main]
 async fn main() {
     match run().await {
-        Ok(_) => println!("Execution terminated with success"),
+        Ok(_) => {
+            println!("Server stopped");
+        }
         Err(e) => {
             eprintln!("An error ocurred executing the HTTP Server");
             eprintln!("{}", e.to_string())
```

Comment:
You don't need the `.to_string()` here (unless it's an inherent method) - the `{}` will use the `Display` trait already.